play-app/src/Components/Main/Main.jsx

```diff
@@ -44,17 +37,17 @@ const Main=()=> {
           </a>
           <a href="" className='product-card'>
               <div className='card-header'>
-                  <img src={playstation5} alt="Playstation" className='product-img'/>
+                  <img src='https://res.cloudinary.com/aniket-singh/image/upload/v1649016035/Images/playstation-5-horizontal-product-shot-01-ps5-en-23nov20_x9ovud.png' alt="Playstation" className='product-img'/>
               </div>
               <div className='card-body'>
                   <div className='card-sale-info f-s font-l flex flex-center m-xs'>on sale</div>
                   <div className='flex f-m font-l' style={{color: 'var(--secondary-color)'}}>Playstation</div>
-                  <div className='flex f-m font-l m-xs' style={{color: 'var(--secondary-color)'}}>रु20000</div>
```

Comment:
Don't use Inline Css. Make it a className.

Reply:
will remove it